javatests/arcs/sdk/examples/unit_testing/ComputePeopleStatsTest.kt

```diff
@@ -0,0 +1,83 @@
+package arcs.sdk.examples.unit_testing
+
+import com.google.common.truth.Truth.assertThat
+import com.google.common.truth.Truth.assertWithMessage
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runBlockingTest
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+/**
+ * Example of particle Unit Testing.
+ */
+@ExperimentalCoroutinesApi
+@RunWith(JUnit4::class)
+class ComputePeopleStatsTest {
+    private lateinit var th: ComputePeopleStatsTestHarness<ComputePeopleStats>;
+
+    @Before
+    fun setUp() = runBlockingTest {
+        th = ComputePeopleStatsTestHarness(this) { ComputePeopleStats(this) }
+    }
```

Comment:
Would this pattern be better as a JUnit Rule?

Reply:
Yes! PTAL.